pkg/test_helpers/http_request_wrapper.go

```diff
@@ -51,25 +52,32 @@ func (hw *HttpWrapper) PerformHttpRequest(address string) (string, error) {
 
 	//address := test.FormUrl(Protocol, DeployName, "0", SubdomainName, ctx1.Namespace, Domain, AddressBit, Port) //nope.
 	// there should be only single address in return in this case.
-    http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 	request, err := http.NewRequest(hw.Method, address, nil)
 	if err != nil {
 		return "", err
 	}
-	request.SetBasicAuth(hw.Password,hw.User)
+	actualPath, _ := url.QueryUnescape(request.URL.Path)
+	request.URL = &url.URL{
+		Scheme: request.URL.Scheme,
+		Host:   request.URL.Host,
+		Opaque: actualPath,
+	}
 	request.Header = *hw.Header
+	request.SetBasicAuth(hw.Password, hw.User)
+	request.URL.User = url.UserPassword("admin", "admin")
```

Comment:
hardcoded user and password. will be better if it is configurable using parameters or config file or at least defined as a constant in the code.

Reply:
@tlbueno in this case, password and user is meaningless (ie, broker would accept anything)